### PR TITLE
Con 3093 migrate stop date

### PR DESCRIPTION
**Pull request by VasanthaKasirajan3008**

### CODE REVIEW

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs [hunk 1]
```diff
@@ -495,12 +497,34 @@ public async Task<IActionResult> ResumeApprenticeship(ResumeRequestViewModel vie
         public async Task<IActionResult> ApprenticeshipDetails(ApprenticeshipDetailsRequest request)
         {
             var viewModel = await _modelMapper.Map<ApprenticeshipDetailsRequestViewModel>(request);
-            if (viewModel.ApprenticeshipStatus == ApprenticeshipStatus.Stopped)
-            {
-                TempData.AddFlashMessage(ApprenticeStoppedMessage, ITempDataDictionaryExtensions.FlashMessageLevel.Success);
-            }
+
+            if (!TempData.ContainsKey(FlashMessageTempDataKey) &&  viewModel.ApprenticeshipStatus == ApprenticeshipStatus.Stopped)
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs:501]: do we always show the flash message, when an apprenticeship is stopped?

**Review by cofaulco — COMMENTED** [review]
1 comment/question

File: src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs (hunk 1 above)
Comment (VasanthaKasirajan3008) [reply src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/ApprenticeController.cs:501]: Yes : we have done as part of AC3 from https://skillsfundingagency.atlassian.net/browse/CON-3172

**Review by cofaulco — APPROVED** [review]

File: src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Validators/EditStopDateViewModelValidatorTests.cs [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+        public void WhenValidatingEditStopDate_ValidateTheNewStopDate(int? newStopMonth, int? newStopYear, bool expectedValid, string errorMessage)
+        {
+            //Arrange
+            var viewModel = _autoFixture.Create<EditStopDateViewModel>();
+            viewModel.NewStopMonth = newStopMonth;
+            viewModel.NewStopYear = newStopYear;            
+
+            //Act
+            var result = _validator.Validate(viewModel);
+            Assert.AreEqual(expectedValid, result.IsValid);
```
Comment (cofaulco) [inline src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Validators/EditStopDateViewModelValidatorTests.cs:80]: this should be below `//Assert

Comment (VasanthaKasirajan3008) [reply src/SFA.DAS.EmployerCommitmentsV2.Web.UnitTests/Validators/EditStopDateViewModelValidatorTests.cs]: done

**Pull request merged**